projects/js-packages/licensing/changelog/update-use-new-react-jsx-transform

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: removed
+Comment: Remove no-longer-needed `@wordpress/babel-plugin-import-jsx-pragma` dep, after switching jest to use the modern JSX transform.
```

Comment:
Super nitpick: technically this (and the others) shouldn't have a comma. Feel free to disregard.

```suggestion
Comment: Remove no-longer-needed `@wordpress/babel-plugin-import-jsx-pragma` dep after switching jest to use the modern JSX transform.
```